atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionManagerTest.java

```diff
@@ -237,7 +237,10 @@ public void registersMetrics() throws InterruptedException {
         TaggedMetricRegistry registry = snapshotTransactionManager.metricsManager.getTaggedRegistry();
         assertThat(registry.getMetrics().keySet().stream().map(MetricName::safeName))
                 .contains(SETUP_TASK_METRIC_NAME)
-                .contains(FINISH_TASK_METRIC_NAME);
+                .contains(FINISH_TASK_METRIC_NAME)
+                .contains("expectations.bytesRead")
+                .contains("expectations.kvsCalls")
+                .contains("expectations.ageMillis");
         assertThat(registry.timer(MetricName.builder()
```

Comment:
I don't see any gain in extracting the magic string to a variable here

Also one of the metrics is missing because it does not get reported if no actual "real" call is made

And, this class mocks everything, doing the plumbing to force it to appear will be ugly NPE hacking until stub/mocks are happy

I believe this + the test file + an RC test is good enough coverage